### PR TITLE
Updating the instructions for the testBeam analysis

### PR DESCRIPTION
**Pull request by yeckang**
The instruction for the testbeam analysis in the README.md is updated.

### CODE REVIEW

File: Analysis/test/simTestBeam.py [hunk 1]
```diff
@@ -107,13 +107,11 @@
...
 process.muonGEMDigis.useDBEMap = process.gemPacker.useDBEMap
 process.muonGEMDigis.keepDAQStatus = True
-process.gemRecHits.gemDigiLabel = cms.InputTag('simMuonGEMDigis')
 
 process.generation_step = cms.Path(process.generator+process.pgen)
 process.simulation_step = cms.Path(process.psim)
 process.digitisation_step = cms.Path(process.mix+process.simMuonGEMDigis)
-#process.digi2raw_step = cms.Path(process.gemRecHits)
-#process.digi2raw_step = cms.Path(process.gemPacker+process.rawDataCollector+process.muonGEMDigis)
+process.digi2raw_step = cms.Path(process.gemPacker+process.rawDataCollector+process.muonGEMDigis)
```
Comment (jshlee) [inline Analysis/test/simTestBeam.py:114]: for sim, this shouldnt be used

File: Simulation/interface/BaseFlatGunProducer.h [hunk 2]
```diff
@@ -0,0 +1,77 @@
+#ifndef BaseFlatGunProducer_H
```
Comment (jshlee) [inline Simulation/interface/BaseFlatGunProducer.h:1]: is this file updated?

**Comment by jshlee** [comment]
did you update the simulation stuff??

**Comment by yeckang** [comment]
I think I messed up while I merging with main branch. I will clean them.

**Pull request merged**